<?xml version="1.0"?>
<launch>
  <include file="$(find racecar_gazebo)/launch/racecar.launch">
	<arg name="world_name" value="racecar_walker"/>
  </include>
  <arg name="wall_config" default="$(find speed_daemons_wall_following)/config/wall_config.yaml" />
  <rosparam file="$(arg wall_config)" command="load"/>

  <node pkg="speed_daemons_wall_following" name="pid_error_node" type="pid_error.py" output="screen"/> 
  <node pkg="speed_daemons_wall_following" name="control_node" type="control.py" output="screen"/>
  <node pkg="speed_daemons_wall_following" name="sim_connector_node" type="sim_connector.py" output="screen"/>
  <node pkg="speed_daemons_wall_following" name="speedDaemons_analysis" type="speedDaemons_analysis.py" output="screen"/>

</launch>
